README.md

```diff
@@ -397,99 +517,193 @@ Also if you like to automatically record the execution time of different cells
-For the case, where there are several OpenMRS systems,
-OpenHIM can be used as a middleware component to integrate/track/log requests from several OpenMRS instances.
+For the case, where there are several OpenMRS systems, OpenHIM can be used as a
+middleware component to integrate/track/log requests from several OpenMRS
+instances.
 
-## 1. Fire up OpenHIM Instance
-After spinning up the Hapi FHIR instance, see details above, you can spin up an OpenHIM instance.
+### 1. Fire up OpenHIM Instance
 
-    `$ docker-compose -f openhim-compose.yaml up # change ports appropriately (optional)` 
-You should be able to access OpenHIM via http://localhost:9000
-Use the default credentials i.e
+After spinning up the Hapi FHIR instance, see details above, you can spin up an
+OpenHIM instance.
 
-    `username : root@openhim.org and password : openhim-password`
+```
+`$ docker-compose -f openhim-compose.yaml up # change ports appropriately (optional)`
+```
 
-Note: You will have problems logging in if your OpenHIM server is still setup to use a self-signed certificate (the default).
-Visit the following link: https://localhost:8093/authenticate/root@openhim.org in Chrome.
+You should be able to access OpenHIM via http://localhost:9000 Use the default
+credentials i.e
 
-You should see a message saying “Your connection is not private”. Click “Advanced” and then click “Proceed”.
+```
+`username : root@openhim.org and password : openhim-password`
+```
+
+Note: You will have problems logging in if your OpenHIM server is still setup to
+use a self-signed certificate (the default). Visit the following link:
+https://localhost:8093/authenticate/root@openhim.org in Chrome.
+
+You should see a message saying “Your connection is not private”. Click
+“Advanced” and then click “Proceed”.
 
-Once you have done this, you should see some JSON text displayed on the screen, you can ignore this and close the page. This will ignore the fact that the certificate is self-signed.
+Once you have done this, you should see some JSON text displayed on the screen,
+you can ignore this and close the page. This will ignore the fact that the
+certificate is self-signed.
 
-Now, you should be able to go back to the OpenHIM console login page and login. This problem will occur every now and then until you load a properly signed certificate into the OpenHIM core server.
+Now, you should be able to go back to the OpenHIM console login page and login.
+This problem will occur every now and then until you load a properly signed
+certificate into the OpenHIM core server.
 
 For the first Login .youll be requested to change your root password!
 
-Then Go to the 'Export/Import' tab and import the default config file under `Utils/openhim-config.json` for a basic configuration.
+Then Go to the 'Export/Import' tab and import the default config file under
+`Utils/openhim-config.json` for a basic configuration.
 
 NB the default configurations have the following default configs:
```

Comment:
I am not sure what NB refers to here but let's leave it as is (after you move this content to a separate file).

Reply:
https://en.wikipedia.org/wiki/Nota_bene though I'd probably just change this to "Note" once I review this section properly. :)